### ISSUE TITLE
Invite after sending first message config

### ISSUE THREAD

**Issue by chagai95** [vector-im/element-web#16317]
**Is your suggestion related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when...

The problem I'm facing is that I go to send a direct message and in other popular messengers I don't invite a person to a room so I sometimes go to their direct chat and take my time until I write the message, sometimes ending up not doing it at all. Here I send an invite straight away and sometimes only a few minutes later the first message...

**Describe the solution you'd like**

The solution would be for example to just not send the invite until the first message is sent.

**Describe alternatives you've considered**

Thought of skipping the whole inviting all together but that's probably another issue... Much simpler to just send the invite only after the first message is sent...

**Additional context**

It's counter intuitive for most people to actually have already contacted the person (sending an invite) just by pressing on direct message even though no message has been sent.

**Comment by SimonBrandner** [comment]
I believe this would be covered by adding invite reason as suggested [here](https://github.com/vector-im/element-web/issues/16300)

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/14736
